tests/Exceptions.py

```diff
@@ -133,6 +133,17 @@ def exceed():
 
         self.assertRaises(github.RateLimitExceededException, exceed)
 
+    def testSecondaryRateLimitExceeded(self):
```

Comment:
There are other messages that indicate secondary rate limit, worth testing them as well? https://github.com/PyGithub/PyGithub/pull/2127#issuecomment-1008842933